packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1526,3 +1526,11 @@ idx.error.code.access_denied.device_assurance.remediation.windows.device_disk_en
 idx.error.code.access_denied.device_assurance.remediation.windows.use_biometric_lock_screen = Enable Windows Hello for the lock screen
 idx.error.code.access_denied.device_assurance.remediation.additional_help_default = For more information, follow the instructions on <$1>the help page</$1> or contact your administrator for help
 idx.error.code.access_denied.device_assurance.remediation.additional_help_custom = For more information, follow the instructions on <$1>your organization's help page</$1> or contact your administrator for help
+# end user remediation for Chrome DTC signals
+idx.error.code.access_denied.device_assurance.remediation.chrome.disk_encrypted = Turn on disk encryption
+idx.error.code.access_denied.device_assurance.remediation.chrome.os_firewall = Turn on your device's firewall
+idx.error.code.access_denied.device_assurance.remediation.chrome.screen_lock_secured = Turn on automatic screen saver and screen locking when idle
+idx.error.code.access_denied.device_assurance.remediation.chrome.upgrade_browser_version = Update Chrome browser to {0}
+idx.error.code.access_denied.device_assurance.remediation.chrome.chromeos.upgrade_os_version = Update to ChromeOS {0}
```

Comment:
Can you add a comment above each of these describing what the variable is? See the examples a few lines up in this file

Reply:
Good point @shawyu-okta , updated